app/helpers/admin_tools.rb

```diff
@@ -0,0 +1,67 @@
+# module AccessLevels
+# 	extend self
+# 	def member
+# 		"member"
+# 	end
+
+# end
+
+module AdminTools
+
+	module MemberManagement
+
+		def add_member(member)
+			default = {"access_level" => "member", "password" => nil}
+			prevent_empty_string_password(member)
+			new_user = User.new(member)
+
+			if new_user.valid?
+				new_user.save
+			else
+				session[:errors] = new_user.errors.messages
+				redirect back
+			end
+			
+		end
+
+		def update_member(member)
+			user = User.find member.delete(:id)
+			user.update_attributes(member)
+		end
+
+		def delete_member(member)
+			user = User.find(member)
+			user.account.manipulations.each(&:destroy)
+			user.account.destroy
```

Comment:
If you're going to delete the account anyway you can setup your account model to delete associated objects like the manipulation records you're manually deleting above.